wwwroot/inc/ajax-interface.php

```diff
@@ -296,4 +296,36 @@ function getNetUsageAJAX()
 	echo getRenderedIPNetCapacity ($net);
 }
 
+
+function getAutocompleteListAJAX()
+{
+	$term = genericAssertion ('term','string0');
+	$realm = genericAssertion ('realm','string');
+
+	if(!$term)
+		return;
+
+	switch($realm)
+	{
+		case 'object':
+			$result = usePreparedSelectBlade("SELECT name from Object where name like '%$term%' order by name");
```

Comment:
Here and below please use a prepared SELECT as `$term` can be any string. Please make SQL keywords uppercase. Please consider LIMIT to avoid returning much more data than the browser can reasonably handle.